automl/google/cloud/automl_v1beta1/proto/io_pb2.py

```diff
@@ -988,8 +988,43 @@
      that wraps a Document proto with input\_config set. Only PDF
      documents are supported now, and each document may be up to 2MB
      large. Currently annotations on documents cannot be specified at
-     import. Any given .JSONL file must be 100MB or smaller.
-
+     import. Any given .JSONL file must be 100MB or smaller. Three sample
+     CSV rows: TRAIN,gs://folder/file1.jsonl
+     VALIDATE,gs://folder/file2.jsonl TEST,gs://folder/file3.jsonl Sample
+     in-line JSON Lines file (presented here with artificial line breaks,
+     but the only actual line break is denoted by :raw-latex:`\n`).: {
+     "text\_snippet": { "content": "dog car cat" }, "annotations": [ {
+     "display\_name": "animal", "text\_extraction": { "text\_segment":
+     {"start\_offset": 0, "end\_offset": 2} } }, { "display\_name":
+     "vehicle", "text\_extraction": { "text\_segment": {"start\_offset":
+     4, "end\_offset": 6} } }, { "display\_name": "animal",
+     "text\_extraction": { "text\_segment": {"start\_offset": 8,
+     "end\_offset": 10} } } ] }:raw-latex:`\n
+           {
+              "text_snippet": {
+                "content": "This dog is good."
+              },
+              "annotations": [
+                 {
+                   "display_name": "animal",
+                   "text_extraction": {
+                     "text_segment": {"start_offset": 5, "end_offset": 7}
+                   }
+                 }
+              ]
+           }` Sample document JSON Lines file (presented here with
+     artificial line breaks, but the only actual line break is denoted by
+     :raw-latex:`\n`).: { "document": { "input\_config": { "gcs\_source":
+     { "input\_uris": [ "gs://folder/document1.pdf" ] } } }
+     }:raw-latex:`\n
+           {
+             "document": {
+               "input_config": {
+                 "gcs_source": { "input_uris": [ "gs://folder/document2.pdf" ]
+                 }
+               }
+             }
+           }`
```

Comment:
More [frustrated novelists jamming hard-ro-render junk in `.proto` docstrings](https://github.com/googleapis/googleapis/blob/32a10f69e2c9ce15bba13ab1ff928bacebb25160/google/cloud/automl/v1beta1/io.proto#L101-L158).

Reply:
It looks like this was supposed to be resolved a few months back: https://github.com/googleapis/gapic-generator/issues/2191, https://github.com/googleapis/artman/pull/535. 

If it takes too long for me to fix manually I'll add a replace to remove it like the one dlp has: https://github.com/googleapis/google-cloud-python/blob/a96fa648a28521906740c0d3efd401e13fbb670a/dlp/synth.py#L250